server/tests/test_api.py

```diff
@@ -41,17 +41,15 @@ def new_run(self, add_tasks=0):
         info="The ultimate patch",
         resolved_base="347d613b0e2c47f90cbf1c5a5affe97303f1ac3d",
         resolved_new="347d613b0e2c47f90cbf1c5a5affe97303f1ac3d",
-        master_sha="347d613b0e2c47f90cbf1c5a5affe97303f1ac3d",
         official_master_sha="347d613b0e2c47f90cbf1c5a5affe97303f1ac3d",
-        merge_base_commit="347d613b0e2c47f90cbf1c5a5affe97303f1ac3d",
         msg_base="Bad stuff",
         msg_new="Super stuff",
         base_signature="123456",
         new_signature="654321",
         base_nets=["nn-0000000000a0.nnue"],
         new_nets=["nn-0000000000a0.nnue", "nn-0000000000a1.nnue"],
         rescheduled_from="653db116cc309ae839563103",
-        tests_repo="https://google.com",
+        tests_repo="https://github.com/sfdeveloper/Stockfish",
```

Comment:
https://github.com/sfdeveloper   ???

Reply:
The schema now checks if `tests_repo` is a syntactically github repo name (but it does not check if the github user exists).  Previously there were adhoc checks at various places.

Reply:
Who is sfdeveloper ?

Reply:
Oh. This is accidental. It doesn't really matter. If you want to we can try to find a non-existing username.